change-beta/@azure-communication-react-7e6e902d-ecc1-4d99-a8e4-6a3abd305ff3.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
```

Comment:
change file should have a stable file and type is `minor` if going from beta -> GA as this now appears in stable builds